worlds/gstla/Rules.py

```diff
@@ -992,7 +992,9 @@ def set_access_rules(world: 'GSTLAWorld'):
         if world.options.reveal_hidden_item == 1:
                 for loc in location_type_to_data[LocationType.Hidden]:
                         #for all hidden items that are not eventype 131 (these are scoopable or cyclone places), we require reveal
-                        if loc.event_type != 131 or loc_names_by_id[loc.ap_id] == LocationName.Daila_Psy_Crystal:
+                        if (loc.event_type != 131 or loc_names_by_id[loc.ap_id] == LocationName.Daila_Psy_Crystal or loc_names_by_id[loc.ap_id] == LocationName.Loho_Mythril_Silver or
```

Comment:
I'd rather it didn't do the map lookup 6 times, but otherwise I have no complaints.

i.e.:

```python
loc_name = loc_names_by_id[loc.ap_id]
if (loc.event_type != 131 or loc_name == LocationName... ):
```